src/chainstate/burn/db/sortdb.rs

```diff
@@ -3022,6 +3232,25 @@ impl<'a> SortitionHandleTx<'a> {
         Ok(())
     }
 
+    /// Insert a pre-stack-stx op
+    fn insert_stack_stx(&mut self, op: &StackStxOp, sort_id: &SortitionId) -> Result<(), db_error> {
```

Comment:
Should this be `insert_pre_stack_stx`, per the comment?

Reply:
Yeah, that's a comment typo

Reply:
Updated in 74da104fb